moveit_ros/benchmarks/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
-find_package(Boost REQUIRED filesystem)
```

Comment:
Is this required by a change in the boost lib or is this something that might have to be fixed for MoveIt 1 as well?

Reply:
I'm not sure, I only encountered https://github.com/ros-planning/moveit2/issues/239 if `date_time` boost component was not explicitly declared. @mikaelarguedas hinted to me that this was the cause before.